tsl/src/compression/compression.c

```diff
@@ -1966,6 +1991,9 @@ create_segment_filter_scankey(RowDecompressor *decompressor, char *segment_filte
 void
 decompress_batches_for_insert(ChunkInsertState *cis, Chunk *chunk, TupleTableSlot *slot)
 {
+	/* Write decompression start marker to logical replication stream if enabled */
```

Comment:
I wonder if this comment adds any value...it seems to me that the method name is already doing a pretty good job in explaining what's happening

Reply:
Fair, I'll remove them